webapp/graphql/Moderation.js

```diff
@@ -82,9 +82,6 @@ export const reportsListQuery = () => {
             name
             disabled
             deleted
-            followedByCount
-            contributionsCount
-            commentedCount
```

Comment:
Awww yeah. That's what I wanted to do in my next PR. I'm going to rebase my code on that.